libs/labelbox/src/labelbox/schema/member.py

```diff
@@ -0,0 +1,433 @@
+    """Represents a members project role
+
+    Args:
+        project_id (str): id of the project you want the member included
+        role (Optional[Role]): Members role for the project. None represents the member having a default role.
+    """
+
+    project_id: str
+    role: Optional[Role] = Field(default=None)
+
+    def __hash__(self) -> int:
+        return self.project_id.__hash__()
+
+    @model_serializer()
+    def serialize_model(self):
+        return {
+            "projectId": self.project_id,
+            "roleId": None if self.role is None else self.role.uid,
+        }
+
+
+class Member(_CamelCaseMixin):
+    """
+    Represents a member in Labelbox.
+
+    Attributes:
+        id (str, frozen): The ID of the member. Defaults to current users id.
+        updated_at (str, frozen): Last time member was updated.
+        created_at (str, frozen): When member was created.
+        email (str, frozen): Email of member.
+        name (str, frozen): Name of the member.
+        nickname (str, frozen): Nickname of the member.
+        picture (str, frozen): Picture of the member.
+        is_viewer (bool, frozen): Indicates if member is a viewer of org.
+        is_external_user (bool, frozen): Indicates if member is a external user of org
+        accessible_project_count (int, frozen): Them amount of projects the user can access
+        project_memberships (Set[ProjectMembership]): The current projects with role the user has access towards
+        can_access_all_projects (bool): If member can access all projects inside org.
+        default_role (Optional[Role]): Shows the members default role. None means the member does not have a default role.
+        user_group_ids (Set[str]): The user group ids the member is associated with.
+        client (Client): The Labelbox client object
+
+    Methods:
+        get(self) -> "Member"
+        update(self) -> "Member"
+        delete(self) -> bool
+        get_user_groups(self, search: str, roles: Optional[list[Role]], group_ids: Optional[list[str]]) -> Iterator["Member"]
+
+    Raises:
+        RuntimeError: If the experimental feature is not enabled in the client.
+    """
+
+    id: str = Field(frozen=True)
+    updated_at: str = Field(default="", frozen=True)
+    created_at: str = Field(default="", frozen=True)
+    email: str = Field(default="", frozen=True)
+    name: Optional[str] = Field(default=None, frozen=True)
+    nickname: Optional[str] = Field(default=None, frozen=True)
+    picture: Optional[str] = Field(default=None, frozen=True)
+    is_viewer: bool = Field(default=False, frozen=True)
+    is_external_user: bool = Field(default=False, frozen=True)
+    accessible_project_count: Optional[int] = Field(default=None, frozen=True)
+    project_memberships: Set[ProjectMembership] = Field(default=set())
+    can_access_all_projects: bool = Field(default=False)
+    default_role: Optional[Role] = Field(default=None)
+    user_group_ids: Set[str] = Field(default=set())
```

Comment:
Avoided making this user group objects since it would sort of get out of control.